packages/core/docs/guide/authentication.md

```diff
@@ -0,0 +1,74 @@
+# What's authentication?
+Authentication is the operation of recognizing the user's identify. It allows us to associate incoming requests with an account or a person. To do that, provided credentials are compared with ones in the database.
+
+The whole logic related to this process can be handled with methods shared by `useUser` [composable](https://v3.vuejs.org/guide/composition-api-introduction.html#basics-of-composition-api).
+
+## Registering a new user
+Creating a user can be done using `register` method.  
+```vue
+<template>
+  <form @submit.prevent="register({ user: form })">
+    <!-- form fields -->
+    <button type="submit" :disabled="loading">Submit</button>
+  </form>
+</template>
+
+<script>
+import { useUser } from '{INTEGRATION}';
+import { ref } from '@vue/composition-api';
+
+export default {
+  setup () {
+    const { register, loading } = useUser();
+    const form = ref(/* object for create user */);
+
+    return {
+      register,
+      loading,
+      form
+    }
+  }
+}
+</script>
+```
+
+## Logging in/out and sessions
+Signing in and out can be done using `login` and `logout` methods. `logout` does not take any arguments.
+```vue
+<template>
```

Comment:
Not so realistic example :d perhaps its better to split it into 2?

Reply:
Ok